GCDWebServer.xcodeproj/project.pbxproj

```diff
@@ -1374,33 +1375,33 @@
 		E2DDD1901BE69404002CE867 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "";
```

Comment:
I'm not sure It should be deleted, default behavior for Xcode is setting them to `""` instead of setting them to nil or deleting. but if you insist, I'll do.
Thanks